.github/workflows/run-danger.yml

```diff
@@ -2,10 +2,12 @@ name: ☢️ Danger
 
 on:
   pull_request:
-    types: [opened, synchronize, edited, review_requested, review_request_removed, labeled, unlabeled, milestoned, demilestoned]
+    types: [opened, reopened, ready_for_review, synchronize, edited, review_requested, review_request_removed, labeled, unlabeled, milestoned, demilestoned]
```

Comment:
Do we need the `review_requested` and `review_request_removed` events? 

I can't come up with something that might have changed in the PR and that wouldn't have already triggered a run that one of those two events would pick up.

Then again, it's cheap to run Dangermattic, so we might as well leave them.

Reply:
Thanks for noticing -- I actually planned to remove them as I figured the "reviewers check" isn't necessary when PRs are auto-assigned to code owners, but forgot to remove the events.
Changed on 856bf8a12410b254a5e9f652c22a80164ffa2de5.